lib/ApplicationFeatures/V8PlatformFeature.cpp

```diff
@@ -81,10 +85,30 @@ static void gcEpilogueCallback(v8::Isolate* isolate, v8::GCType type,
   size_t usedHeapSize = h.used_heap_size();
   size_t stillFree = heapSizeLimit - usedHeapSize;
 
+  if (now - v8g->_lastMaxTime > 10*1000) {
+    v8g->_heapMax = heapSizeAtStart;
+    v8g->_heapLow = heapSizeAtStop;
+    v8g->_countOfTimes = 0;
+    v8g->_lastMaxTime = now;
+  } else {
+    v8g->_countOfTimes++;
+    if (heapSizeAtStart > v8g->_heapMax) {
+      v8g->_heapMax = heapSizeAtStart;
+      v8g->_lastMaxTime = now;
```

Comment:
is it correct to bump the timestamp here? then at least description is not right for the following case:
1s: heap usage 10M
2s: heap usage 11M
3s: heap usage 12M
etc.
This can carry on for more than 10 seconds.
But it will always bump the timestamp and increase `_countOfTimes`, so `_countOfTimes` and the other values reported are not for a 10 second interval, but for an interval of arbitrary and unknown length.